proj/settings.py

```diff
@@ -119,7 +119,7 @@
     },
     "JAVASCRIPT": {
         "caps": {
-            "source_filenames": ("caps/js/main.js",),
+            "source_filenames": ("caps/js/main.js", "charting/js/altair-fallback.js"),
```

Comment:
Plus it would be good to only load this extra JS on pages that actually need it (rather than concatenating it all into the same `main.js` that gets loaded on every page).